enterprise/frontend/src/embedding-sdk-bundle/lib/sdk-question/load-question.ts

```diff
@@ -26,16 +26,21 @@ export const loadQuestionSdk =
     Required<Pick<SdkQuestionState, "question">> &
       Pick<SdkQuestionState, "originalQuestion" | "parameterValues">
   > => {
-    const questionId = initQuestionId === "new" ? undefined : initQuestionId;
+    const isNewQuestion = initQuestionId === "new";
+    const questionId = isNewQuestion ? undefined : initQuestionId;
 
-    const { card, originalCard } = await resolveCards({
+    const { card: resolvedCard, originalCard } = await resolveCards({
       cardId: questionId ?? undefined,
       options,
       dispatch,
       getState,
       deserializedCard,
     });
 
+    const card = isNewQuestion
+      ? { ...resolvedCard, creationType: "custom_question" }
```

Comment:
We've been doing this like forever. This isn't new.
https://github.com/metabase/metabase/blob/3a85044d81e885246a6e0fc5aa1c2b5717df8949/frontend/src/metabase/dashboard/components/QuestionPicker/actions.ts#L19-L22

It's just that we never set `createType` to SDK question and https://github.com/metabase/metabase/pull/63855 surfaces this problem